sessions/react-effects-and-localstorage/journal-app-localstorage/src/App.js

```diff
@@ -0,0 +1,88 @@
+import { useState } from "react";
+import "./App.css";
+import EntriesSection from "./components/EntriesSection";
+import EntryForm from "./components/EntryForm";
+import Footer from "./components/Footer";
+import Header from "./components/Header";
+import { nanoid } from "nanoid";
+
+const initialEntries = [
+  {
+    id: 1000,
+    date: "Feb 5, 2025",
+    motto: "We are in a state of chaos",
+    notes:
+      "Today I learned about React State. It was fun! I can't wait to learn more.",
+  },
+  {
+    id: 999,
+    date: "Feb 4, 2025",
+    motto: "Props, Props, Props",
+    notes:
+      "Today I learned about React Props. Mad props to everyone who understands this!",
+  },
+  {
+    id: 998,
+    date: "Feb 3, 2025",
+    motto: "How to nest components online fast",
+    notes:
+      "Today I learned about React Components and how to nest them like a pro. Application design is so much fun!",
+  },
+  {
+    id: 997,
+    date: "Feb 2, 2025",
+    motto: "I'm a React Developer",
+    notes: "My React-ion when I learned about React: 😍",
+  },
+];
+
+function App() {
+  const [entries, setEntries] = useState(initialEntries);
+  const [showFavorites, setShowFavorites] = useState(false);
```

Comment:
nitpick 1: We teach that boolean should ideally start with `is`, `has`, `should`.
`show` is not far off, but we could maybe do something like this.

```js
  const [isShowFavorites, setIsShowFavorites] = useState(false);
```

That name is very very bad though…


so maybe a string filter? It looked at it for a while and think it's not too much more difficult…

```suggestion
  const [filter, setFilter] = useState("all"); // "all" or "favorites"
```